packages/workflow/src/Interfaces.ts

```diff
@@ -1898,3 +1912,15 @@ export interface IExceutionSummaryNodeExecutionResult {
 		description?: string;
 	}>;
 }
+
+export interface ResourceMapperFields {
+	fields: Array<{
+		id: string;
+		displayName: string;
+		match: boolean;
+		defaultMatch: boolean;
+		required: boolean;
+		display: boolean;
+		type?: string;
```

Comment:
always string or we could set other types here that would be corectly renderd in NDV?